ephios/plugins/federation/models.py

```diff
@@ -0,0 +1,90 @@
+import dataclasses
+
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+from ephios.api.models import AccessToken, Application
+from ephios.core.models import AbstractParticipation, Event, Qualification
+from ephios.core.signup.participants import AbstractParticipant
+
+
+class FederatedGuest(models.Model):
+    name = models.CharField(max_length=255)
+    url = models.URLField()
+    access_token = models.ForeignKey(AccessToken, on_delete=models.CASCADE)
+    client_id = models.CharField(max_length=255)
+    client_secret = models.CharField(max_length=255)
+
+
+class FederatedHost(models.Model):
```

Comment:
Reading the model names causes a lot of confusion in me. Maybe we should name all models with clear terminologic prefix like `IncomingFederatedUser` and `OutgoingFederatedEventShare` and `OutgoingFederationPartnership`.